### PR TITLE
Update "No Plots" section  message

### LINKED ISSUE #4462: Improve message when plots are created for the first time
When a user adds plots to `dvc.yaml` and runs an experiment for the first time, `plots diff` sends `File Not Found` errors until the plots are created. How this looks currently:

https://github.com/iterative/vscode-dvc/assets/43496356/cb5cda4a-1135-4529-abc8-3ad690b3bee7

Can we improve what the extension shows to better explain what's going on? 

Followup to #4412

### PR DESCRIPTION
**Pull request by julieg18**
## Demo

~https://github.com/iterative/vscode-dvc/assets/43496356/2e854bb5-587c-46ad-aa14-727d8c6200f9~

image

image

image

image

Fixes #4462

### CODE REVIEW

File: webview/src/plots/components/emptyState/AddPlots.tsx [hunk 1]
```diff
@@ -10,7 +10,10 @@ export const AddPlots: React.FC<AddPlotsProps> = ({
   hasUnselectedPlots
 }: AddPlotsProps) => (
   <div>
-    <p>No Plots to Display</p>
+    <p>
+      No Plots to Display (Plots take longer to appear when being created for
+      the first time.)
```
Comment (julieg18) [inline webview/src/plots/components/emptyState/AddPlots.tsx:15]: Any other ideas for a better message?

**Comment by mattseddon** [comment]
Defer to @shcheklein

**Review by sroy3 — APPROVED** [review]
Just my personal opinion, I'd add a new line for the new text. Shorter lines are easier to read.

**Comment by shcheklein** [comment]
Thanks @julieg18 !

Let's be super explicit here:

No plots or data to display.
Note: If you have some experiments selected that just started you might need to wait while data to plot is produced. The screen will be updated automatically if plots are setup correctly in this case.

Also, let's please review some other places where we have a similar message.

**Comment by shcheklein** [comment]
Look good to me, thanks @julieg18 !

**Review by mattseddon — APPROVED** [review]

**Pull request merged**